alex/applications/PublicTransportInfoCS/hdc_policy.py

```diff
@@ -175,16 +176,39 @@ def get_da(self, dialogue_state):
         # reset all slots depending on changed slots
         self.reset_on_change(dialogue_state, changed_slots)
 
+        c = {
```

Comment:
It stands for 'condition'. I didn't want a long identifier. But maybe 'fact' or 'prop' (proposition) would be more intuitive?